src/components/Slider/Slider.tsx

```diff
@@ -1,8 +1,11 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
```

Comment:
Do these two lines need to be there? I don't get any errors when running `npm run lint`

Reply:
Ah yes, after reinstalling node modules the errors disappeared, will remove this